jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/codeeditor/JmixCodeEditor.java

```diff
@@ -65,79 +68,140 @@ protected void initComponent() {
         }
 
         setInvalid(false);
+        setPrintMarginColumn(PRINT_MARGIN_COLUMN_DEFAULT_VALUE);
+        setFontSize(FONT_SIZE_DEFAULT_VALUE);
+
         addValueChangeListener(e -> validate());
         addClientValidatedEventListener(e -> validate());
     }
 
+    /** @see JmixCodeEditor#setHighlightActiveLine(boolean)
+     *
+     * @return Visibility of highlighting the active line
+     */
     @Synchronize(PROPERTY_HIGHLIGHT_ACTIVE_LINE)
     public boolean isHighlightActiveLine() {
         return getElement().getProperty(PROPERTY_HIGHLIGHT_ACTIVE_LINE, true);
     }
 
+    /**
+     * Sets the highlight of the line the cursor is on
```

Comment:
1. For all setters add info about the default value.
2. The `.` is missing at the end of the sentence. 
3. No, sentence after `@param` and `@return` doesn't need a `.` at the end
4. And add `@param` for all setters.